app/javascript/components/cloud-volume-actions-form/create-backup.schema.js

```diff
@@ -0,0 +1,43 @@
+import { componentTypes } from '@data-driven-forms/react-form-renderer';
+
+function createBackupSchema() {
+  const fields = [{
+    component: componentTypes.SUB_FORM,
+    title: __('Basic Information'),
+    id: 'basic-information',
+    name: 'basic-information',
+    fields: [
+      {
+        component: componentTypes.TEXT_FIELD,
+        id: 'name',
+        name: 'name',
+        label: __('Backup Name'),
+        maxLength: 50,
+      }],
+  },
+  {
+    component: componentTypes.SUB_FORM,
```

Comment:
In current master, i am not seeing these fields for some volumes. Could you check?